src/utils/gaa.js

```diff
@@ -479,8 +491,45 @@ export class GaaMeteringRegwall {
       regwallContainer.remove();
     }
   }
+
+  /**
+   * Returns true if the URL contains fresh Google Article Access (GAA) params.
+   * @private
+   * @return {boolean}
+   */
+  static urlContainsFreshGaaParams_() {
+    const params = parseQueryString(
+      GaaMeteringRegwall.location_.search.split('?')[1]
```

Comment:
We might want to check for only the first '?' in the string and consider everything after that valid: https://tools.ietf.org/html/rfc3986#section-3.4

Reply:
It looks like the parseQueryString function already does some splitting for you: https://github.com/subscriptions-project/swg-js/blob/ad4118da72087b1b6002e8fc2740945bc281fcfe/src/utils/url.js#L163

Reply:
Nice catch!